main.cpp

```diff
@@ -882,8 +899,10 @@ class state_controller { // Variables Implemented
                 LOG("plugged to manual charger\n");
                 set_new_state(POWER_STATE::MANUAL_CHARGE);
             } else if (!charge_guard_asserted && ac.is_docked() && bmu.is_chargable()) {
-                LOG("docked to auto charger\n");
-                set_new_state(POWER_STATE::AUTO_CHARGE);
+                if(ac.is_charger_ready() == true){
+                    LOG("docked to auto charger\n");
+                    set_new_state(POWER_STATE::AUTO_CHARGE);
+                }
```

Comment:
スペースの入れ方を他と揃える。
Align the way you put spaces with others.
```suggestion
                if (ac.is_charger_ready() == true) {
                    LOG("docked to auto charger\n");
                    set_new_state(POWER_STATE::AUTO_CHARGE);
                }
```

Reply:
修正してコミットしました。また、別の箇所もスペースの入れ方が違ったので、同様に修正しました。

Fixed and committed. Another section was also corrected in the same manner, as it was spaced incorrectly.